catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc

```diff
@@ -38,6 +39,15 @@ Additionally, a user can save individual records that were returned by the query
 they can be referenced in the future. The user can also save the search criteria to a workspace so
 the query can be run again.
 
+==== Turning on Search UI
```

Comment:
✏️ I think you'll want to use the same verbiage as we use in our other installation sections, which I think might be "Installing ...."

Reply:
fixed